todo_list/todo_list_app/models.py

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Try deleting this file and running the project. It should run due to the init imports.